kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/sqlbuilder/AbstractPipelineSQLBuilder.java

```diff
@@ -190,4 +190,10 @@ public String buildSplitByPrimaryKeyRangeSQL(final String schemaName, final Stri
         return String.format("SELECT MAX(%s),COUNT(*) FROM (SELECT %s FROM %s WHERE %s>=? ORDER BY %s LIMIT ?) t",
                 quotedUniqueKey, quotedUniqueKey, getQualifiedTableName(schemaName, tableName), quotedUniqueKey, quotedUniqueKey);
     }
+    
+    @Override
+    public String buildSelectOffsetSQL(final String schemaName, final String tableName) {
+        String qualifiedTableName = getQualifiedTableName(schemaName, tableName);
+        return String.format("SELECT * FROM %s LIMIT ? OFFSET ?", qualifiedTableName);
```

Comment:
It's better not use LIMIT and OFFSET. Since records ordering are not guaranteed, it might get duplicated records or miss some records?